app.js

```diff
@@ -16,38 +16,34 @@
 
 const express = require('express');
 const statusCodes = require('http-status-codes');
+
 const errorTracker = require('./routes/error-tracker');
 const querystring = require('./utils/requests/query-string');
 const parseErrorHandling = require('./utils/requests/parse-error-handling');
-const json = require('./utils/requests/json');
 
 const app = express();
-const port = parseInt(process.env.PORT, 10) || 8080;
+const jsonParser = express.json({
+  limit: 10 * 1024 /* 10kb */,
+  type: () => true, // Attempt to allow any content-type.
+});
 
 app.set('etag', false);
 app.set('trust proxy', true);
 app.set('query parser', querystring.parse);
+// Handle BodyParser PayloadTooLargeError errors
+app.use(parseErrorHandling);
 
-app.get(['/readiness_check', '/liveness_check', '/_ah/health'], (req, res) => {
-  res.sendStatus(statusCodes.OK);
+app.get(['/', '/r'], (req, res) => {
```

Comment:
We could actually just delete this code path. It was for when we migrated from query params to POST, but there shouldn't be any releases still doing query params.

Reply:
Neato! Will do

Reply:
Done.